src/state.rs

```diff
@@ -232,56 +236,28 @@ impl State {
             });
         }
 
-        {
-            let mut guard = self.active_capella_org.lock().unwrap();
-            *guard = Some(active);
-        }
-
-        Ok(())
-    }
-
-    pub fn set_active_capella_org_id(&mut self, id: String) -> Result<(), ShellError> {
-        let active = match self.active_capella_org_name() {
-            Some(a) => a,
-            None => {
-                return Err(ShellError::GenericError {
-                    error: "No active Capella organization set".to_string(),
-                    msg: "".to_string(),
-                    span: None,
-                    help: None,
-                    inner: Vec::new(),
-                })
+        if let Some(org) = self.capella_orgs.get(&active) {
```

Comment:
We should probably combine this with the above check as a single match statement or similar.

Reply:
Done